### ISSUE TITLE
Hyperparamaters

### ISSUE THREAD

**Issue by regilim** [WuJie1010/Facial-Expression-Recognition.Pytorch#49]
Sorry, I'm new to Deep Learning.
May I know what are the hyperparameters you used to get VGG19  of Test_acc： 94.646% and Resnet18 of Test_acc： 94.040%
Mine seem to always be getting an average accuracy of 88% after 10 folds for both.
I used lr 0.01, bs 128, epoch 40, learning_rate_decay_start = 20, learning_rate_decay_every = 1 and learning_rate_decay_rate = 0.8

**Comment by WuJie1010** [comment]
please try again~I will make it reproducible via random seed if i have time.
